symengine/integer.h

```diff
@@ -9,8 +9,7 @@
 
 #include <symengine/basic.h>
 #include <symengine/number.h>
-#include <gmpxx.h>
```

Comment:
I tried compiling symengine on a system with only gmp without C++ library. So, this shouldn't be here